pandas/tests/groupby/test_groupby.py

```diff
@@ -2994,16 +3021,25 @@ def test_non_cython_api(self):
         assert_frame_equal(result, expected)
 
         # describe
-        expected = DataFrame(dict(B=concat(
-            [df.loc[[0, 1], 'B'].describe(), df.loc[[2], 'B'].describe()],
-            keys=[1, 3])))
-        expected.index.names = ['A', None]
+        expected = pd.concat([(df[df.A == 1].B
+                                            .describe()
+                                            .to_frame()
```

Comment:
In this test, the `result = df.groupby('A').describe().unstack()` after `unstack()` was added to `groupby.describe()`. Shouldn't `expected` follow an independent path to the `result`?